assets/evaluation_results/boolq_gpt_35_turbo_0301_question_answering/spec.yaml

```diff
@@ -0,0 +1,32 @@
+type: evaluationresult
+name: boolq_gpt_35_turbo_0301_question_answering
+version: 1.0.0
+display_name: boolq_gpt_35_turbo_0301_question_answering
+description: gpt-35-turbo-0301 run for boolq dataset
+dataset_family: boolq
+dataset_name: boolq
+
+model_name: gpt-35-turbo-0301
+model_version: "301"
```

Comment:
Not critical - but these versions strings really should be "0301" and "0314" to match the way that OAI/AOAI versions these models

Reply:
Will change them in the next release.